apps/site/next.config.mjs

```diff
@@ -62,101 +53,46 @@ const nextConfig = {
   output: ENABLE_STATIC_EXPORT ? 'export' : undefined,
   // This configures all the Next.js rewrites, which are used for rewriting internal URLs into other internal Endpoints
   // This feature is not supported within static export builds, hence we pass an empty array if static exports are enabled
-  rewrites: !ENABLE_STATIC_EXPORT ? rewrites : undefined,
+  rewrites: ENABLE_STATIC_EXPORT ? undefined : rewrites,
   // This configures all Next.js redirects
-  redirects: !ENABLE_STATIC_EXPORT ? redirects : undefined,
+  redirects: ENABLE_STATIC_EXPORT ? undefined : redirects,
   // We don't want to run Type Checking on Production Builds
   // as we already check it on the CI within each Pull Request
   typescript: { ignoreBuildErrors: true },
   // We don't want to run ESLint Checking on Production Builds
   // as we already check it on the CI within each Pull Request
   // we also configure ESLint to run its lint checking on all files (next lint)
   eslint: { dirs: ['.'], ignoreDuringBuilds: true },
-  // Adds custom WebPack configuration to our Next.hs setup
-  webpack: function (config, { webpack }) {
-    // Next.js WebPack Bundler does not know how to handle `.mjs` files on `node_modules`
-    // This is not an issue when using TurboPack as it uses SWC and it is ESM-only
-    // Once Next.js uses Turbopack for their build process we can remove this
-    config.module.rules.push({
-      test: /\.m?js$/,
-      type: 'javascript/auto',
-      resolve: { fullySpecified: false },
-    });
-
-    // Tree-shakes modules from Sentry Bundle
-    config.plugins.push(new webpack.DefinePlugin(SENTRY_EXTENSIONS));
-
-    // Ignore Sentry's Critical Dependency from Open Telemetry
-    // (which is genuinely a cause of concern, but there is no work around at the moment)
-    config.ignoreWarnings = [
-      {
-        module: /@opentelemetry\/instrumentation/,
-        message: /Critical dependency/,
-      },
-    ];
-
-    return config;
-  },
   experimental: {
-    // Some of our static pages from `getStaticProps` have a lot of data
-    // since we pass the fully-compiled MDX page from `MDXRemote` through
-    // a page's static props.
-    largePageDataBytes: 128 * 100000,
+    // Run CSS optimizations/bundle minification with Critters
+    optimizeCss: true,
```

Comment:
have you tested lightingcss ? experimental features ?

Reply:
If you read LightningCSS experimental docs, you would know it does not support PostCSS plugins at the moment.

Reply:
optimizeCss option is not about faster CSS compilations but reducing CSS bundle size.

Reply:
okay got it !